docs/learn/learn-runtime-upgrades.md

```diff
@@ -56,56 +45,55 @@ Although upgrading your nodes is generally not necessary to follow an upgrade, w
-to upgrade within a specific time frame to ensure continued sync with their node. It is essential to
-check the release notes, starting with the upgrade priority and acting accordingly.
+to upgrade their clients within a specific time frame, for example if a release includes breaking
+changes to networking. It is essential to check the release notes, starting with the upgrade
+priority and acting accordingly.
 
-For general infrastructure providers, aside from following the Polkadot releases and upgrading in a
-timely manner, somethings to keep an eye out on is updating your parsing logic and acknowledging
-changes to the available RPC clients, such as the
+General infrastructure providers, aside from following the Polkadot releases and upgrading in a
+timely manner, should monitor changes to runtime events and auxilliary tooling, such as the
 [Substrate API Sidecar](https://github.com/paritytech/substrate-api-sidecar).
 
+Transactions constructed for runtime `n` will not work for runtimes `>n`. If a runtime upgrade
+occurs before broadcasting a priorly constructed transaction, you will need to reconstruct it with
+the appropriate runtime version and corresponding metadata.
+
 ### For [Nominators](../maintain/maintain-guides-how-to-nominate-polkadot.md)
 
 Runtime upgrades don't require any actions by a nominator, though, it is always encouraged to keep
 up-to-date and participate with the latest runtime upgrade motions and releases, while keeping an
 eye on how the nodes on the network are reacting to a new upgrade.
 
-### For [Wallets](../build/build-wallets.md)
-
-Wallets should look out for updates to the transaction lifecycle, such as tx broadcasting. An
-example would be constructing a transaction with a new spec, "n", but broadcasting it with a spec
-" > n " - transactions could be propagated, or even rejected by peers on the network.
-
 ## Monitoring Changes
 
-Using a Polkadot blockchain explorer, you can monitor the chain for on-chain changes. The following
-steps relate to [subscan](https://polkadot.subscan.io/).
-
-> In general, an action has two components: the module (such as `democracy`) and the event (such as
-> `Started`).
-
-Monitor the chain for:
-
-1. `democracy(Started)` events and log `index` and `blockNumber`. Get
-   `pallets/democracy/storage/ReferendumInfoOf?key1=index&at=blockNumber` from `Sidecar` to get the
-   referendum info. It should have a status of `Ongoing`. Find the ending block number (`end`) and
-   the enactment `delay` (delay), where the execution block number will be `end + delay`.
+You can monitor the chain for upcoming upgrades. The client release notes include the hashes of any
+proposals related to any on-chain upgrades for easy matching. Monitor the chain for:
 
+1. `democracy(Started)` events and log `index` and `blockNumber`. This indicates that a referendum
+   has started (although does not mean that it is a runtime upgrade). Get the referendum info\*; it
```

Comment:
"(although does not mean that it is a runtime upgrade)" -> "(although, this does not mean that it is a runtime upgrade)"

Reply:
No, there's no a reason to introduce an undeclared "this".